test_dagmc_stats.py

```diff
@@ -53,3 +53,12 @@ def test_get_entityset_ranges():
     volume_range = my_core.get_entities_by_type_and_tag(root_set, types.MBENTITYSET, dagmc_tags['geom_dim'], [3])
     assert(volume_range == entityset_ranges['Volumes'])
     
+
+def test_get_surfaces_per_volume():
```

Comment:
In this model, we know that there are 3 volumes: 2 cubes and a sphere, and that the cubes have 6 surfaces per volume and the sphere has 1.  We should be able to test the actual values of this data, not just the length of the list.